test/functional/api/v2/template_controller_test.rb

```diff
@@ -26,7 +26,7 @@ class TemplateControllerTest < ::ActionController::TestCase
           FactoryBot.create(:provisioning_template, :name => 'export_test_template')
           post :export, params: { "repo" => tmpdir, "filter" => "^export_test_template", "negate" => true, "metadata_export_mode" => "keep" }
           assert_response :success
-          refute_includes(Dir.entries("#{tmpdir}/provisioning_templates/provision"), 'export_test_template.erb')
```

Comment:
Hmm.. Is this necessary?

Reply:
I guess, I've skipped this while rebasing, lemme fix it!